docs/README.md

```diff
@@ -52,6 +52,3 @@ Aditionally to the trigger configuration, you must create the message center tem
 	"addToCartURL": "add?sku=1&qty=1&seller=1&sku=2&qty=1&seller=1"
 }
 ```
-
-
-The `addToCartURL` variable is formatted to work with the [cart URL](https://help.vtex.com/tutorial/how-to-assemble-the-cart-url--u3Tj5wagnukYwG84IQU06). Its purpose is to send the user back to the checkout with the items in the abandoned cart.
```

Comment:
@elkyndiosa here we should use the current description

Reply:
Done!